packages/babel-parser/src/plugins/flow.js

```diff
@@ -2392,7 +2393,10 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       refNeedsArrowPos?: ?Pos,
     ): N.Expression {
       let jsxError = null;
-      if (tt.jsxTagStart && this.match(tt.jsxTagStart)) {
+      if (
+        this.hasPlugin("jsx") &&
```

Comment:
I had to replace this, as `tt.jsxTagStart` is always set in our tests, even if jsx plugin not loaded. Yeahh monkeypatching :)

Reply:
Yeah, that are the most hard to debug test failures :man_facepalming: